lua/luasnip/init.lua

```diff
@@ -55,6 +55,8 @@ end
 
 local function get_context(snip)
 	return {
+		id = snip.id,
+		meta_data = snip.meta_data,
```

Comment:
Oh, could you rebase on master? this is now a bit different, the `get_context` used by `available` is configurable now, I don't think this is necessary anymore.

Reply:
I can rebase ofc but I don't understand the issue - could you explain further. I am on latest master locally.

Reply:
Are you sure you're on latest master? I merged a PR this morning, which touches `available`, and I don't see its changes here

Reply:
Or I'm completely missing something xD

Reply:
Ima look into this tomorrow. Im in sweden time zone

Reply:
> I merged a PR this morning, which touches `available`, and I don't see its changes here

You were right ofc. I must have pulled right before you pushed...

Reply:
I think that available should be modified again, so that one can pass it 
1. "everything" -> return every single snippet
2. table list of filetypes -> return all requested snippets
3. current version -> return only available to current buffer.

Reply:
Ahh unfortunate